core/src/main/java/org/mule/service/processor/ServiceProcessorExecutor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.service.processor;
+
+import org.mule.VoidMuleEvent;
+import org.mule.api.MessagingException;
+import org.mule.api.MuleEvent;
+import org.mule.api.processor.MessageProcessor;
+import org.mule.execution.MessageProcessorExecutionTemplate;
+import org.mule.processor.BlockingProcessorExecutor;
+
+import java.util.List;
+
+/**
+ * {@link org.mule.processor.BlockingProcessorExecutor} specifically for use with instances of {@link org.mule.api.service.Service}
+ */
+public class ServiceProcessorExecutor extends BlockingProcessorExecutor
```

Comment:
Do we really want to add tests for services when it's deprecated. Especially given this isn't new code, it's just move from one place to another.